src/index.js

```diff
@@ -1,18 +1,11 @@
 /* eslint-disable */
```

Comment:
- [ ] Please kindly do not disable linters, fix linters manually helps programmers to write clean and clear codebase 😁 😉